operations/template/app.tf

```diff
@@ -53,6 +52,8 @@ resource "azurerm_linux_web_app" "sftp" {
     health_check_path                 = "/health"
     health_check_eviction_time_in_min = 5
 
+    container_registry_use_managed_identity = true
```

Comment:
Down below on what is currently line `84` and `85`, we have...
```
DOCKER_REGISTRY_SERVER_USERNAME = azurerm_container_registry.registry.admin_username
DOCKER_REGISTRY_SERVER_PASSWORD = azurerm_container_registry.registry.admin_password
```

Can we remove those lines and this continues to work?

Reply:
We can try it and find out 😅